src/hyperloglog.rs

```diff
@@ -152,15 +152,18 @@ where
 
     /// Adds an element to the HyperLogLog.
     pub fn add(&mut self, obj: &T) {
-        let mut hasher = self.buildhasher.build_hasher();
-        obj.hash(&mut hasher);
-        let h: u64 = hasher.finish();
+        self.add_hashed(self.buildhasher.hash_one(obj));
+    }
 
-        // split h into:
+    /// Adds an already-hashed element to the HyperLogLog
+    ///
+    /// Note: Make sure to use the same hasher as the rest of the HyperLogLog when hashing values on your own
+    pub fn add_hashed(&mut self, hashed_value: u64) {
```

Comment:
why is this public?

Reply:
If I have a value that's already been hashed, there is no way for me to insert it otherwise. I can't unhash it..